src/components/Reactions/EmojiReactionBubble.tsx

```diff
@@ -82,7 +83,8 @@ function EmojiReactionBubble(
             accessible
             dataSet={{[CONST.SELECTION_SCRAPER_HIDDEN_ELEMENT]: true}}
         >
-            <Text style={[styles.emojiReactionBubbleText, StyleUtils.getEmojiReactionBubbleTextStyle(isContextMenu)]}>{emojiCodes.join('')}</Text>
+            {/* TODO: investigate why emojis are moved up on iOS on the new architecture */}
```

Comment:
Please don't leave TODOs in the code. Just create an issue and link it to here with a PR comment.

Reply:
Issue is created already so I just added a comment linking to it